src/api/spec/controllers/webui/comments_controller_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+RSpec.describe Webui::CommentsController do
+  let(:user) { create(:confirmed_user, login: 'luck') }
+  let(:user_tom) { create(:confirmed_user, login: 'tom') }
+  let(:admin) { create(:admin_user, login: 'Admin') }
+  let(:project) { create(:project, name: 'openSUSE') }
+  let(:package) { create(:package, name: 'osc', project: project) }
+  let(:bs_request) { create(:bs_request, description: "Please take this", creator: user.login) }
+  let(:comment) { create(:comment_project, user: user) }
+  let(:tom_comment) { create(:comment_project, user: user_tom) }
+
+  describe 'POST #create' do
+    before do
+      login user
+    end
+
+    context 'saving a comment of a project' do
+      before do
+        post :create, params: { comment: { body: 'This project is AWESOME!', commentable_type: project.class, commentable_id: project.id } }
+      end
+
+      it { expect(flash[:notice]).to eq('Comment was successfully created.') }
+    end
+
+    context 'saving a comment of a package' do
+      before do
+        post :create, params: { comment: { body: 'Wonderful Package', commentable_type: package.class, commentable_id: package.id } }
+      end
+
+      it { expect(flash[:notice]).to eq('Comment was successfully created.') }
+    end
+
+    context 'saving a comment of a request' do
```

Comment:
Hm, what about saving of other class does not work? E.g. with current implementation 

```
post :create, params: { comment: { body: 'Wonderful Package', commentable_type: Distribution, commentable_id: 42 } }
```

would work.